keps/sig-node/4817-resource-claim-device-status/README.md

```diff
@@ -0,0 +1,730 @@
+
+### Version Skew Strategy
+
+This feature affects only the kube-apiserver, so there is no issue with version
+skew with other Kubernetes components.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: DRAResourceClaimDeviceStatus
+  - Components depending on the feature gate: kube-apiserver
+- [ ] Other
+  - Describe the mechanism:
+  - Will enabling / disabling the feature require downtime of the control
+    plane?
+  - Will enabling / disabling the feature require downtime or reprovisioning of
+    a node?
+
+###### Does enabling the feature change any default behavior?
+
+No
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes, with no side effects except for missing the new field `Devices` in
+`ResourceClaim.Status`.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+The field will be available again for read and write, but this feature will 
+not guarantee to keep the values written before the feature has been disabled.
+DRA-Drivers will then have to write again the `ResourceClaim.Status.Devices`.
+
+###### Are there any tests for feature enablement/disablement?
+
+Enablement/disablement of this feature is tested as part of the integration
+tests.
+
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+Enabling the feature will enable the field to be written and therefore invoke 
+validation of the field. Since the feature does nothing more than allow storage 
+of this data, the only way it can really fail is to crash in validation. It 
+will not affect running workloads.
+
+###### What specific metrics should inform a rollback?
+
+N/A
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+N/A
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No
+
+### Monitoring Requirements
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Check the `ResourceClaim.Status.Devices`.
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [ ] Events
+  - Event Reason: 
+- [x] API .status
+  - Condition name: 
+  - Other field: `ResourceClaim.Status.Devices`
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+N/A
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- [x] Metrics
+  - Metric name: `resourceclaim_status_devices_update_attempts_total` ; `resourceclaim_status_devices_update_failures_total`
```

Comment:
nit (won't block merge): can you make this a yaml-style list? Also note the first one above in the "how can an operator determine if the feature is in use"

Reply:
Done